application/values.yaml

```diff
@@ -634,3 +634,44 @@ prometheusRule:
     #       for: 1m
     #       labels:
     #         severity: critical  
+##########################################################
+# External Secrets
+##########################################################
+externalSecret:
```

Comment:
so, we are taking input a list or a map? can a value in one environment be overridden by a value in another environment?

Reply:
we are taking as map now. it can be overridden